Lib/types.py

```diff
@@ -143,6 +143,35 @@ def _calculate_meta(meta, bases):
                         "of the metaclasses of all its bases")
     return winner
 
+
+def get_original_bases(cls, /):
+    """
+    Return the objects in the bases list in the class's definition before
+    they could have been modified by ``__mro_entries__``. This is useful for
+    introspecting Generics.
```

Comment:
PEP 257: The first line of a docstring should be a standalone summary of the function that exists on the same line as the opening quote.

```suggestion
    """Return the class's "original" bases prior to modification by `__mro_entries__`.
    
    This is useful for introspecting Generics.
```